src/locale/de.json

```
@@ -30,7 +30,51 @@
     "areYouSure": "Sind Sie absolut sicher?",
     "updateButton": "Aktualisieren",
     "updateConfirmation": "Sind Sie sicher, dass Sie ein Software-Update starten möchten? Dies kann mehrere Minuten dauern. Sie können keinen Stream starten, bis das Update abgeschlossen ist. Der Encoder wird nach einem erfolgreichen Upgrade kurzzeitig getrennt. Entfernen Sie während der Aktualisierung niemals die Stromversorgung und setzen Sie den Encoder nicht zurück. Wenn der Encoder mit einer Batterie betrieben wird, stellen Sie sicher, dass diese vollständig geladen ist.",
-    "streamingMessage": "Ihre Übertragung nutzt {usingNetworksCount} Netzwerke mit einer Verzögerung von {srtLatency} ms"
+    "streamingMessage": "Ihre Übertragung nutzt {usingNetworksCount} Netzwerke mit einer Verzögerung von {srtLatency} ms",
```

Biome (2.1.2)

[error] The key streamingMessage was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)